src/main/java/com/twilio/oai/TwilioRubyGenerator.java

```diff
@@ -53,6 +54,8 @@ public void processOpenAPI(final OpenAPI openAPI) {
         setGemName("");
         this.libFolder = "";
         directoryStructureService.configure(openAPI);
+        Map<String, PathItem> pathMap = openAPI.getPaths();
+        RubyApiResourceBuilder.configureAdditionalProps(pathMap, domain, directoryStructureService);
```

Comment:
Does this have to be static?

Reply:
This was intentionally made static to avoid making another constructor in `RubyApiResourceBuilder`.

Reply:
Should this function not be `directoryStructureService.configureAdditionalProps` instead?

Reply:
Makes sense! I can refactor this.

Reply:
Done.